### PR TITLE
Add information about ActivityStarter special support for Parceler

### PR DESCRIPTION
**Pull request by MarcinMoskala**

### CODE REVIEW

**Comment by johncarl81** [comment]
Cool!  Do you want to update the [website](http://parceler.org/) as well?  You can do this via a PR on https://github.com/johncarl81/parceler-site.

**Comment by MarcinMoskala** [comment]
Sure :D 
https://github.com/johncarl81/parceler-site/pull/7

File: README.asciidoc [hunk 1]
```diff
@@ -79,6 +79,7 @@ This wrapping and unwrapping technique plays well with the Intent Factory patter
 In addition, Parceler is supported by the following libraries:
 
  * http://androidtransfuse.org/documentation.html#parcel[Transfuse] - Allows `@Parcel` annotated beans to be used with the `@Extra` injection.
+ * https://github.com/MarcinMoskala/ActivityStarter/wiki/Parceler-Arg-Converter-usage[ActivityStarter] - Uses ParcelarArgConverter to allow to use Parceler objects as an arguments passed to Activities (Fragments, Services etc.).
```
Comment (johncarl81) [inline README.asciidoc]: Make ParcelarArgConverter code formatted (use ``` character).

Comment (johncarl81) [inline README.asciidoc]: Also, move this down to the bottom of the list.

**Comment by MarcinMoskala** [comment]
Please, hold it. I will add native support for Parceler

**Comment by johncarl81** [comment]
I'm ok merging this for now (I'll move the link down too).  Let me know how native support goes.

**Pull request merged**